src/Makefile.qt.include

```diff
@@ -296,12 +296,7 @@ QML_RES_FONTS = \
 
 QML_QRC_CPP = qml/qrc_bitcoin.cpp
 QML_QRC = qml/bitcoin_qml.qrc
-QML_RES_QML = \
-  qml/components/BlockCounter.qml \
-  qml/components/ConnectionOptions.qml \
-  qml/controls/OptionButton.qml \
-  qml/pages/initerrormessage.qml \
-  qml/pages/stub.qml
+$(eval QML_RES_QML = $(shell $(RCC) --list $(QML_QRC)))
```

Comment:
Maybe
```suggestion
QML_RES_QML :=  $(shell $(RCC) --list $(QML_QRC))
```
?

Reply:
My suggestion does not work due to Automake [limitations](https://www.gnu.org/software/automake/manual/html_node/Conditional-Sources.html).